### PR TITLE
Buildwork to remove dir.props files

### PR DESCRIPTION
**Pull request by davidwrighton**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -179,8 +179,7 @@ restore_optdata()
 
 generate_event_logging_sources()
 {
-    __OutputDir=$1
-    __OutputEventingDir="$__OutputDir/Eventing"
+    __OutputEventingDir=$1
```
Comment (jashook) [inline build.sh]: Does build.sh need to use the empty props as well?

File: src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```
Comment (jashook) [inline src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj]: This seems a little unnatural. It seems like this should be changed to an sdk style project? @sbomer any reason you know of that would inhibit this?

File: src/.nuget/Microsoft.NETCore.ILDAsm/Microsoft.NETCore.ILDAsm.pkgproj [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```
Comment (jashook) [inline src/.nuget/Microsoft.NETCore.ILDAsm/Microsoft.NETCore.ILDAsm.pkgproj]: Same issue as the ilasm proj.

File: src/.nuget/Microsoft.NETCore.Jit/Microsoft.NETCore.Jit.pkgproj [hunk 4]
```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```
Comment (jashook) [inline src/.nuget/Microsoft.NETCore.Jit/Microsoft.NETCore.Jit.pkgproj]: Same issue as the ilasm proj

File: src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds [hunk 5]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.Build.Traversal">
+<Project>
   <Import Project="Microsoft.NETCore.Native.props" />
-
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="Sdk.props" Sdk="Microsoft.Build.Traversal" />
```
Comment (jashook) [inline src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds]: What is the reason for importing the props and targets manually instead of letting the project do the import? Do we have conflicting imports?

If so can you comment them here?

File: src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.pkgproj [hunk 6]
```diff
@@ -3,13 +3,13 @@
 
   <Import Project="Microsoft.NETCore.Native.props" />
 
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```
Comment (jashook) [inline src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.pkgproj]: Same issue as the ilasm pkgproj

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj [hunk 7]
```diff
@@ -3,7 +3,7 @@
 
   <Import Project="Microsoft.NETCore.Runtime.CoreCLR.props" />
 
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```
Comment (jashook) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: Same issue as the ilasm pkgproj

File: src/.nuget/Microsoft.NETCore.TestHost/Microsoft.NETCore.TestHost.pkgproj [hunk 8]
```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```
Comment (jashook) [inline src/.nuget/Microsoft.NETCore.TestHost/Microsoft.NETCore.TestHost.pkgproj]: Same issue as the ilasm pkgproj

File: src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```
Comment (jashook) [inline src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj]: Same issue as the ilasm pkgproj

File: src/System.Private.CoreLib/System.Private.CoreLib.csproj [hunk 10]
```diff
@@ -419,7 +417,7 @@
   </ItemGroup>
   <Import Project="shared\System.Private.CoreLib.Shared.projitems" />
   <PropertyGroup>
-    <CheckCDefines Condition="'$(CheckCDefines)'==''">true</CheckCDefines>
+    <CheckCDefines Condition="'$(CheckCDefines)'=='' And '$(PYTHON)' != ''">true</CheckCDefines>
```
Comment (jashook) [inline src/System.Private.CoreLib/System.Private.CoreLib.csproj]: It seems like silently avoiding checks if python is not installed is incorrect. My opinion would be to allow it to fail, and we create a follow up future work item to convert the check to msbuild.

File: tests/build.proj [hunk 11]
```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="dir.props" />
+  <Import Project="Directory.Build.props" />
```
Comment (jashook) [inline tests/build.proj]: I think the correct solution for build.proj is to convert it to an sdk style project.

File: tests/src/dirs.proj [hunk 12]
```diff
@@ -1,5 +1,5 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="dir.props" />
+  <Import Project="Directory.Build.props" />
```
Comment (jashook) [inline tests/src/dirs.proj]: This probably should also be an sdk style project.

File: tests/src/runtest.proj [hunk 13]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="src\dir.props" />
+  <Import Project="Directory.Build.props" />
```
Comment (jashook) [inline tests/src/runtest.proj]: This also should probably be an sdk style project.

File: src/System.Private.CoreLib/System.Private.CoreLib.csproj (hunk 10 above)
Comment (davidwrighton) [reply src/System.Private.CoreLib/System.Private.CoreLib.csproj]: Mmm. I don't need to make this change at all. Its really just a convenience as I prefer for it to be possible to compile major projects in the tree without specifying any special parameters to the msbuild command. I'll remove this adjustment.

File: src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds (hunk 5 above)
Comment (davidwrighton) [reply src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds]: Directory.Build.props depends on the properties set in Microsoft.NETCore.Native.props. I've tried some build mysticism to have the Directory.Build.props pull in this props file if its needed. We'll see if it works.

File: tests/build.proj (hunk 11 above)
Comment (davidwrighton) [reply tests/build.proj]: Converting .proj files into sdk style projects does odd things. If we need to do that, it should be a separate change.

File: tests/src/dirs.proj (hunk 12 above)
Comment (davidwrighton) [reply tests/src/dirs.proj]: Converting .proj files into sdk style projects does odd things. If we need to do that, it should be a separate change.

File: tests/src/runtest.proj (hunk 13 above)
Comment (davidwrighton) [reply tests/src/runtest.proj]: Converting .proj files into sdk style projects does odd things. If we need to do that, it should be a separate change.

File: build.cmd [hunk 14]
```diff
@@ -430,7 +434,7 @@ REM ===
 REM =========================================================================================
 
 set __IntermediatesIncDir=%__IntermediatesDir%\src\inc
-set __IntermediatesEventingDir=%__IntermediatesDir%\Eventing
+set __IntermediatesEventingDir=%__ArtifactsIntermediatesDir%\Eventing\%__BuildArch%\%__BuildType%
```
[the remark this answers is not in the document]
Comment (davidwrighton) [reply build.cmd]: This matches with where the Arcade logic puts stuff. I'm not changing that.

File: src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj (hunk 2 above)
Comment (AaronRobinsonMSFT) [reply src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj]: This is a `*.pkgproj` so I don't think there is any SDK support. What this really should be is a blank/custom style. Remove the `ToolsVersion` and `xmlns` attributes.

File: src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds (hunk 5 above)
Comment (AaronRobinsonMSFT) [reply src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds]: I would prefer setting the appropriate properties that drive the `Traversal` logic so they can be included naturally if possible.

File: dir.common.props [hunk 15]
```diff
@@ -95,4 +95,41 @@
...
+  </PropertyGroup>
+
+  <!-- Common properties -->
+  <PropertyGroup>
+
+    <!-- This name is used to create a GIT repository URL https://github.com/dotnet/$(GitHubRepositoryName) used to find source code for debugging -->
+    <GitHubRepositoryName Condition="'$(GitHubRepositoryName)' == ''">coreclr</GitHubRepositoryName>
+
+    <Platform Condition="'$(Platform)' == ''">$(BuildArch)</Platform>
+    <Platform Condition="'$(Platform)' == 'amd64'">x64</Platform>
```
Comment (elinor-fung) [inline dir.common.props]: This is should already be handled above (lines 23 and 41)

File: dir.common.props [hunk 16]
```diff
@@ -95,4 +95,41 @@
...
+
+    <!-- Default to portable build if not explicitly set -->
+    <PortableBuild Condition="'$(PortableBuild)' == ''">true</PortableBuild>
+
+    <OverrideRestoreOutputPath>true</OverrideRestoreOutputPath>
+
+    <!-- If true, indicates that this is not an officially supported release -->
+    <!-- It is important to flip this to false in official release branches -->
+    <!-- Keep it in sync with PRERELEASE in clrdefinitions.cmake -->
+    <IsPrerelease>true</IsPrerelease>
```
Comment (elinor-fung) [inline dir.common.props]: This is already set in src/Directory.Build.props. If it needs to be set at this higher level instead, we should remove it from that file (which should already include this file by way of the root Directory.Build.props).

Comment (davidwrighton) [reply dir.common.props]: It isn't needed at a higher level, but it might (and was set at the root level by the way the pre-existing dir.props worked. If we ever have tests that depend on prerelease features, they would need this. I've simply removed it from src/Directory.Build.props which should be fine.

File: src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds (hunk 5 above)
Comment (elinor-fung) [reply src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds]: Couldn't you just rename `Microsoft.NETCore.Native.props` to `Directory.Build.props` and have it set the required properties before chaining in the `Directory.Build.props` from the parent folder?

File: tests/Directory.Build.props [hunk 17]
```diff
@@ -45,13 +46,14 @@
...
   </PropertyGroup>
 
-  <!-- Provides properties for dependency versions and configures dependency verification/auto-upgrade. -->
-  <Import Project="$(ProjectDir)..\dependencies.props" />
-
   <!-- Add test-specific package restore sources. -->
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json;
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
```
Comment (sbomer) [inline tests/Directory.Build.props]: Why were these added, and would it be possible to put them in NuGet.Config instead? Arcade is recommending to move away from `RestoreSources`: https://github.com/dotnet/arcade/blob/f8fc65bce78b79d00c66cb22ff658fb3a3e46dfb/Documentation/RestoreSourcesUpdateStatus.md.

File: EmptyProps.props [hunk 18]
```diff
@@ -0,0 +1,8 @@
+
+<Project>
+<!-- This file is intentionally empty. It is copied by build.cmd into the bin directory as both Directory.Build.props
+     and Directory.Build.targets to prevent CMake built projects from dependending on the Directory.Build infrastructure
+     in the root of the source tree. In particular this was necessary to compile DacTableGen, which is currently compiled
+     against the desktop framework.
```
Comment (elinor-fung) [inline EmptyProps.props]: Can we configure the generated projects in CMake such that they set `ImportDirectoryBuildProps` and `ImportDirectoryBuildTargets` to false?

File: src/.nuget/Directory.Build.props [hunk 19]
```diff
@@ -2,10 +2,12 @@
 <Project>
 
   <!-- Packaging projects (.pkgproj) are non-SDK-style, so they need to directly import Directory.Build.props -->
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))"
-          Condition="'$(MSBuildProjectExtension)' == '.pkgproj'" />
+  <Import Project="../Directory.Build.props" />
   <Import Project="packaging.props" />
 
+  <!-- Allow specialization of the packaging by creating a props file of the same name of the project immediately next to the project -->
+  <Import Condition="Exists('$(MSBuildProjectDirectory)/$(MSBuildProjectName).props')" Project="$(MSBuildProjectDirectory)/$(MSBuildProjectName).props"/>
```
Comment (elinor-fung) [inline src/.nuget/Directory.Build.props]: Did you have a chance to try renaming the Microsoft.NETCore.Native.props instead? I think it should work and it would be more in line with msbuild's existing system for customization.

File: EmptyProps.props (hunk 18 above)
Comment (davidwrighton) [reply EmptyProps.props]: No

File: src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds (hunk 5 above)
Comment (davidwrighton) [reply src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds]: Hmm, that would probably work. I was under the impression that Directory.Build.props was always looked for in the directory above the project, but it seems reasonable that it might support in the same directory logic.

File: tests/Directory.Build.props (hunk 17 above)
Comment (davidwrighton) [reply tests/Directory.Build.props]: Well, we weren't using nuget.config in our test builds at all. But the move to unify the properties resulted in an empty RestoreSources at this point in the build. This then added a single entry, and that wasn't sufficient for restore to work. I've moved the test build process to use nuget.config and that appears to work fine. Nice catch.

File: dependencies.props [hunk 20]
```diff
@@ -11,9 +11,6 @@
     </RestoreSources>
```
[the remark this answers is not in the document]
Comment (davidwrighton) [reply dependencies.props]: Yes we can. See my response to @sbomer

File: dependencies.props [hunk 21]
```diff
@@ -11,9 +11,6 @@
...
   </PropertyGroup>
 
-  <!-- [ARCADE REMOVE] Versions.props is imported by Arcade SDK targets when building with Arcade -->
-  <Import Project="eng/Versions.props" Condition="'$(ArcadeBuild)' != 'true'" />
-
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
```
[the remark this answers is not in the document]
Comment (davidwrighton) [reply dependencies.props]: To be honest, I have no idea what any of the dependency stuff does. I don't think these changes I've made don't actually add these to our build, they have always been there.

File: dir.common.props (hunk 15 above)
Comment (davidwrighton) [reply dir.common.props]: Looks like it. Removed

File: dir.common.props [hunk 22]
```diff
@@ -95,4 +95,41 @@
...
+    <!-- This name is used to create a GIT repository URL https://github.com/dotnet/$(GitHubRepositoryName) used to find source code for debugging -->
+    <GitHubRepositoryName Condition="'$(GitHubRepositoryName)' == ''">coreclr</GitHubRepositoryName>
+
+    <Platform Condition="'$(Platform)' == ''">$(BuildArch)</Platform>
+    <Platform Condition="'$(Platform)' == 'amd64'">x64</Platform>
+
+    <!-- Default to portable build if not explicitly set -->
+    <PortableBuild Condition="'$(PortableBuild)' == ''">true</PortableBuild>
+
+    <OverrideRestoreOutputPath>true</OverrideRestoreOutputPath>
```
[the remark this answers is not in the document]
Comment (davidwrighton) [reply dir.common.props]: I dunno. There don't appear to be any uses of it. So I've removed it.

File: src/.nuget/Directory.Build.props (hunk 19 above)
Comment (davidwrighton) [reply src/.nuget/Directory.Build.props]: Yeah, that looks to work.

**Comment by davidwrighton** [comment]
@elinor-fung could you take a look and confirm that your concerns have been addressed?

File: src/.nuget/Directory.Build.targets [hunk 23]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (davidwrighton) [reply src/.nuget/Directory.Build.targets]: Everything is now hierarchical. Yes, its not ideal to add pollution to the .builds files, but its consistent, and non-harmful.

File: src/.nuget/Directory.Build.props [hunk 24]
```diff
@@ -2,8 +2,7 @@
 <Project>
 
   <!-- Packaging projects (.pkgproj) are non-SDK-style, so they need to directly import Directory.Build.props -->
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))"
-          Condition="'$(MSBuildProjectExtension)' == '.pkgproj'" />
+  <Import Project="../Directory.Build.props" />
   <Import Project="packaging.props" />
```
[the remark this answers is not in the document]
Comment (davidwrighton) [reply src/.nuget/Directory.Build.props]: Everything is now hierarchical. Yes, its not ideal to add pollution to the .builds files, but its consistent, and non-harmful.

File: build-test.cmd [hunk 25]
```diff
@@ -324,7 +324,7 @@ for /l %%G in (1, 1, %__NumberOfTestGroups%) do (
 
     if not "%__CopyNativeTestBinaries%" == "1" (
         REM Disable warnAsError - coreclr issue 19922
-        set __MSBuildBuildArgs=!__ProjectDir!\tests\build.proj -warnAsError:0 !__Logging! !TargetsWindowsMsbuildArg! !__msbuildArgs! !__PriorityArg! !__UnprocessedBuildArgs! /p:CopyNativeProjectBinaries=!__CopyNativeProjectsAfterCombinedTestBuild!
+        set __MSBuildBuildArgs=!__ProjectDir!\tests\build.proj -warnAsError:0 /nodeReuse:false !__Logging! !TargetsWindowsMsbuildArg! !__msbuildArgs! !__PriorityArg! !__UnprocessedBuildArgs! /p:CopyNativeProjectBinaries=!__CopyNativeProjectsAfterCombinedTestBuild!
```
[the remark this answers is not in the document]
Comment (elinor-fung) [reply build-test.cmd]: We should make a note to set nodeReuse to false in configure-toolset.ps1/sh (separately from this change).

File: build.cmd [hunk 26]
```diff
@@ -302,6 +302,7 @@ if /i %__BuildType% NEQ Release set __RestoreOptData=0
 
 set "__BinDir=%__RootBinDir%\Product\%__BuildOS%.%__BuildArch%.%__BuildType%"
 set "__IntermediatesDir=%__RootBinDir%\obj\%__BuildOS%.%__BuildArch%.%__BuildType%"
+set "__ArtifactsIntermediatesDir=%__ProjectDir%\artifacts\obj\"
```
[the remark this answers is not in the document]
Comment (elinor-fung) [reply build.cmd]: Are native ones still in `bin/obj`? Might be worth commenting.

File: dependencies.props (hunk 21 above)
[the remark this answers is not in the document]
Comment (elinor-fung) [reply dependencies.props]: Yeah, just noticed that it is always being included now (by the top-level .props), which doesn't seem to hurt anything, but might be good to clean up. Maybe @sbomer knows from his work to consume Microsoft.DotNet.VersionTools for updating published versions?

File: eng/run-test-job.yml [hunk 27]
```diff
@@ -355,3 +355,12 @@ jobs:
           scenarios:
           - jitelthookenabled
           - jitelthookenabled_tiered
+
+    # Publish Logs
+    - task: PublishBuildArtifacts@1
+      displayName: Publish Logs
+      inputs:
+        pathtoPublish: $(Build.SourcesDirectory)/bin/Logs
+        artifactName: ${{ format('TestLogs_run_test_job_{0}{1}_{2}_{3}_{4}_{5}_{6}', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig, parameters.managedTestBuildOsGroup, parameters.managedTestBuildOsSubgroup, parameters.testGroup) }}
```
Comment (elinor-fung) [inline eng/run-test-job.yml]: TestRunLogs_ as the prefix? Tiny bit shorter...

File: sync.cmd [hunk 28]
```diff
@@ -21,7 +21,6 @@ REM x gets the next argument, and y gets all remaining arguments after the first
 FOR /f "tokens=1*" %%x IN ("%*") DO (
     set param=%%x
     if /i "!param!" == "-p"                     (set __MSBuildArgs=!__MSBuildArgs! .\build.proj /p:RestoreDuringBuild=true /t:Sync)
-    if /i "!param!" == "-ab"                    (set __MSBuildArgs=!__MSBuildArgs! .\src\syncAzure.proj)
```
Comment (elinor-fung) [inline sync.cmd]: Can we just kill this entire file?

File: tests/build.proj [hunk 29]
```diff
@@ -1,14 +1,14 @@
...
 
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
```
Comment (elinor-fung) [inline tests/build.proj]: Remove? Pretty sure `ToolsDir` was for BuildTools

File: tests/scripts/scripts.csproj [hunk 30]
```diff
@@ -27,7 +27,6 @@
     <RuntimeIdentifiers>win7-x86;win7-x64</RuntimeIdentifiers>
     <IsTestProject>false</IsTestProject>
   </PropertyGroup>
-  <Import Project="$(MSBuildThisFileDirectory)..\src\dir.targets" />
```
Comment (elinor-fung) [inline tests/scripts/scripts.csproj]: Does this need to explicitly pull in a Directory.Build.targets?

**Comment by elinor-fung** [comment]
@davidwrighton yeah, I think I'm generally okay with the product - honestly, did not look hard at the tests...

File: eng/run-test-job.yml (hunk 27 above)
Comment (davidwrighton) [reply eng/run-test-job.yml]: ok

File: sync.cmd (hunk 28 above)
Comment (davidwrighton) [reply sync.cmd]: yes

File: dependencies.props (hunk 21 above)
[the remark this answers is not in the document]
Comment (sbomer) [reply dependencies.props]: I'm not familiar with them either - they were used for buildtools dependency flow, and we can probably get rid of most of them now that we're on arcade. I'd need to go through them one-by-one to see which ones are still required for VersionTools.
